example/components/ChartDisplay.js

```diff
@@ -1,15 +1,23 @@
 'use client';
-import AkvoCharts from 'akvo-charts';
+import { Bar, Doughnut, Line, Pie } from 'akvo-charts';
 import { useChartContext } from '../context/ChartContextProvider';
 
 const ChartDisplay = () => {
```

Comment:
Resize the charts to full width when we hide the right panel.

Reply:
Resolved with this commit [4a818bb](https://github.com/akvo/akvo-charts/pull/25/commits/4a818bb6dc60c8dc8f1179c7a166c0db0713a61a)